.gitignore.template

```diff
@@ -1,5 +0,0 @@
-.gitignore
```

Comment:
The reason to not check in a gitignore file is to avoid forcing the same gitignore on everyone.

Reply:
It's actually helpful to have a shared `.gitignore` for all the contributors, and quite surprising that it's missing 🙂
If a private file is needed, one should use `.git/info/exclude`.
If something is missing from the shared file - please let me know.